packages/tracing/src/browser/metrics.ts

```diff
@@ -154,6 +182,14 @@ export class MetricsInstrumentation {
             ...(entry.size && { elementSize: entry.size }),
             value: entry.startTime,
           };
```

Comment:
Let's get rid of `this._lcp` in favor of having it all under `this._measurements`; there's no need to store LCP twice.

Reply:
Removed in https://github.com/getsentry/sentry-javascript/pull/2909/commits/688c985302792b19078294003f2cbc56a04a2eda